pwiz_tools/Skyline/TestRunnerLib/RunTests.cs

```diff
@@ -1029,13 +1029,19 @@ public void Log(string info, params object[] args)
         {
             lock (_logLock)
             {
-                Console.Write(info, args);
-                Console.Out.Flush(); // Get this info to TeamCity or SkylineTester ASAP
-                if (_log != null)
-                {
-                    _log.Write(info, args);
-                    _log.Flush();
-                }
+                Log(_log, info, args);
+            }
+        }
+
+        [StringFormatMethod("info")]
+        public static void Log(StreamWriter log, string info, object[] args)
```

Comment:
Maybe a comment that this function is not threadsafe like the non-static version?